data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/IsmPolicyManagement.java

```diff
@@ -142,14 +149,33 @@ public CreateIndexRequest getCreateIndexRequest(final String indexAlias) {
     }
 
     private String retrievePolicyJsonString(final String fileName) throws IOException {
+        if (fileName.startsWith(S3_PREFIX)) {
+            return retrievePolicyJsonStringFromS3(fileName);
+        } else {
+            return retrievePolicyJsonStringFromFile(fileName);
+        }
+    }
+
+    private String retrievePolicyJsonStringFromS3(final String fileName) throws IOException {
+        final StringBuilder policyJsonBuffer = new StringBuilder();
+        final FileReader s3FileReader = new S3FileReader(s3Client);
+        try (final InputStream inputStream = s3FileReader.readFile(fileName);
+             final BufferedReader reader = new BufferedReader(new InputStreamReader(Objects.requireNonNull(inputStream)))) {
+            reader.lines().forEach(line -> policyJsonBuffer.append(line).append("\n"));
```

Comment:
@dlvenable do wee need to use Jackson here as well? Something like this instead of BufferedReader and appending each line
```
final Map<String, Object> stringObjectMap = new ObjectMapper().readValue(inputStream, new TypeReference<Map<String, Object>>() {});
jsonPolicy = new ObjectMapper().writeValueAsString(stringObjectMap);
```